core/src/discord.cpp

```diff
@@ -56,6 +62,21 @@ QJsonDocument Discord::constructModcallJson(const QString &f_name, const QString
     return QJsonDocument(l_json);
 }
 
+QJsonDocument Discord::constructBanJson(const QString &f_ipid, const QString &f_moderator, const QString &f_duration, const QString &f_reason, const int &f_banID)
+{
+    QJsonObject l_json;
+    QJsonArray l_array;
+    QJsonObject l_object {
+        {"color", "13312842"},
+        {"title", "User has been banned by " + f_moderator},
```

Comment:
```suggestion
        {"title", "Ban issued by " + f_moderator},
```
"User" is nonspecific and awkward